.ci/lib.sh

```diff
@@ -30,6 +30,45 @@ info() {
 	echo -e "INFO: $*"
 }
 
+# If CI is defined, execute the command arguments under chronic
+# with a 'ticker heartbeat echo' so the CI system does not timeout
+# due to inactivity.
+# Otherwise, run the commands directly.
+ci_chronic() {
+	if [ "$CI" == true ]; then
+		# If we ever get the kata_chronic.sh script landed in the
+		# .ci dir, then call that here instead of open coding it.
+		local cmdLine=$@
+
+		eval chronic "${cmdLine[@]}" &
+		local cmdPid="$!"
+
+		(
+			local sleepval=10
+			local echoval=60
+			local count=0
+			while true; do
+				printf ".";sleep ${sleepval};
+				((count+=${sleepval}))
+				printf $count
+				((count%${echoval} == 0)) && printf ":\n" && count=0
+			done
+		)&
+
+		local printerPid="$!"
+
+		wait "$cmdPid"
+		local ret=$?
+		printf "\n"
+		kill "$printerPid"
```

Comment:
it's me again :) 
So after some online research I figured out that you could add a `wait "$printerPid"` right after the `kill "$printerPid"` to silence the `Terminated` message.